gtk/src/lib.rs

```diff
@@ -20,67 +20,81 @@
 //!
 //! - [General `GLib` family types and object system overview](../glib/index.html)
 //!
-//! - [GTK+ documentation](https://www.gtk.org/docs/)
+//! - [GTK documentation](https://www.gtk.org/docs/)
 //!
-//! # Hello World
+//! ## "Hello, World!" example program
 //!
 //! ```no_run
 //! use gtk::prelude::*;
-//! use gtk::{ButtonsType, DialogFlags, MessageType, MessageDialog, Window};
+//! use gtk::{ApplicationBuilder, ApplicationWindowBuilder};
 //!
 //! fn main() {
-//!     if gtk::init().is_err() {
-//!         println!("Failed to initialize GTK.");
-//!         return;
-//!     }
-//!     MessageDialog::new(None::<&Window>,
-//!                        DialogFlags::empty(),
-//!                        MessageType::Info,
-//!                        ButtonsType::Ok,
-//!                        "Hello World").run();
+//!     let app = ApplicationBuilder::new()
+//!         .application_id("org.example.HelloWorld")
+//!         .build();
+//!
+//!     app.connect_activate(|app| {
+//!         // We create the main window.
+//!         let win = ApplicationWindowBuilder::new()
+//!             .application(app)
+//!             .default_width(320)
+//!             .default_height(200)
+//!             .title("Hello, World!")
+//!             .build();
+//!
+//!         // Don't forget to make all widgets visible.
+//!         win.show_all();
+//!     });
+//!
+//!     app.run();
 //! }
 //! ```
 //!
-//! # Initialization
+//! ## Initialization
 //!
-//! GTK+ needs to be initialized before use by calling [`init`](fn.init.html) or
-//! [`Application::new`](struct.Application.html#method.new). You only need to
-//! do it once and there is no 'finalize'.
+//! GTK needs to be initialized before use by calling [function@init] or by
+//! creating an [struct@Application]. You only need to do it once and there
+//! is no 'finalize'.
```

Comment:
> You only need to do it once and there is no 'finalize'.

I don't know what this means (even before your changes) :) @GuillaumeGomez?

Reply:
I meant that you don't need to free it. (Yep, can definitely be improved haha)

Reply:
The sentence is gone now as it makes not much sense :) Memory is managed automatically in the bindings everywhere.

Reply:
Yes! This is rust here! :laughing: